tests/test_host/test_docker_management.py

```diff
@@ -1,24 +0,0 @@
-"""
```

Comment:
So I removed this. The GitHub CI doesn't have docker for macos and windows doesn't run linux images. So we could create a new test, but given that this isn't the most useful test I don't think its worth it.